lua/lualine/components/filetype.lua

```diff
@@ -30,6 +30,7 @@ function M:apply_icon()
   local ok, devicons = pcall(require, 'nvim-web-devicons')
   if ok then
     local f_name, f_extension = vim.fn.expand '%:t', vim.fn.expand '%:e'
+    f_extension = f_extension ~= '' and f_extension or vim.api.nvim_buf_get_option(0, 'filetype')
```

Comment:
I think we should use `vim.bo` instead of `api.nvim_buf_get_option` since that's what used in rest of the code base.

Reply:
Sure